### ISSUE TITLE
[Excel.Files] Headers ignored during 'Create Worksheet'

### ISSUE THREAD

**Issue by Noordsestern** [robocorp/rpaframework#51]
Hello,

following example: I have a dictionary mapping worksheet names and tables. Those tables include headers. When running this keyword, headers are missing in the final worksheet.
```
Save values in '${ausgabedatei}'
    create workbook    ${OUTPUT_DIR}/${ausgabedatei}
    FOR    ${name}    ${tabelle}     IN    &{PRODUKTWERTE}
        create worksheet    ${name}    ${tabelle}
    END
    save workbook
    [teardown]    close workbook
```

Regards,
Markus

**Comment by osrjv** [comment]
Hi,

Sorry for the late reply, somehow I missed the notification for this. There is an option to write headers, but it's apparently not exposed in the create worksheet keyword.

Can you try if this works for you:
``
FOR    ${name}    ${tabelle}     IN    &{PRODUKTWERTE}
    Create worksheet    ${name}
    Append rows to worksheet    ${tabelle}    header=${TRUE}
END
``

**Comment by osrjv** [comment]
Fixed in commit cab50cfe8b3c986b5694c31e1aee2f90c4aabb6d
